crates/gitbutler-core/src/virtual_branches/virtual.rs

```diff
@@ -66,7 +66,7 @@ pub struct VirtualBranch {
     pub upstream_name: Option<String>, // the upstream branch where this branch will push to on next push
     pub base_current: bool, // is this vbranch based on the current base branch? if false, this needs to be manually merged with conflicts
     pub ownership: BranchOwnershipClaims,
-    pub updated_at: Duration,
+    pub updated_at: u128,
```

Comment:
I assume that these values are for dates we don't get from a commit, so one would expect people to have their clocks in the current time.

But then again, I wonder why risk it, and not use a type that can handle any time a system could use.
Maybe it's about serialization here? But if it was, couldn't that be handled in the code the serializes and deserializes?

Ah, I understand, this was also reverted to the original type.